src/Tools/Source/RunTests/RunTests.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="$(MicrosoftTeamFoundationServerClientVersion)" />
```

Comment:
Is the root reason caused by `Microsoft.TeamFoundationServer.Client`?

Reply:
Yes